### ISSUE TITLE
Still Private DNS not working

### ISSUE THREAD

**Issue by moon-web28** [varunsridharan/pi-hole-android-private-dns#16]
I'm using Pi-Hole v5.11.4 and installed using the below command in my ubuntu machine but still Private DNS not working
```
sudo wget https://raw.githubusercontent.com/GhostlyCrowd/pi-hole-android-private-dns/main/pi-hole5.sh
sudo bash pi-hole5.sh {domain_name} {email_for_letsencrypt}
```
Can you please tell me how I can debug it?
netstat -tulpn | grep LISTEN result
![image](https://user-images.githubusercontent.com/71059061/214036850-a9de6375-695a-4dba-bccf-5243508cdbec.png)

**Comment by varunsridharan** [comment]
https://blog.svarun.dev/configure-pi-hole-with-dns-over-tls-private-dns

<blockquote><img src="https://hashnode.com/utility/r?url=https%3A%2F%2Fcdn.hashnode.com%2Fres%2Fhashnode%2Fimage%2Fupload%2Fv1606305042766%2FH8dCWUS5M.jpeg%3Fw%3D1200%26h%3D630%26fit%3Dcrop%26crop%3Dentropy%26auto%3Dcompress%2Cformat%26format%3Dwebp%26fm%3Dpng" width="48" align="right"><div><img src="https://cdn.hashnode.com/res/hashnode/image/upload/v1604463472654/zbdO5sfnz.png?auto=compress,format&format=webp&fm=png" height="14"> VS Writings</div><div><strong><a href="https://blog.svarun.dev/configure-pi-hole-with-dns-over-tls-private-dns">Configure Pi-Hole With DNS over TLS - [ Private DNS ]</a></strong></div><div>When Google officially launched Android 9 Pie, which includes a slew of new features around digital well-being, security, and privacy. 
If you’ve poked around the network settings on your phone, you may have noticed a new settings called Private DNS ...</div></blockquote>

**Comment by moon-web28** [comment]
@varunsridharan I Already did all the steps, still not working, when I add my domain in Private DNS it's throws Couldn't Connect error and the internet is stop working

**Comment by IsHacker003** [comment]
What domain do you need to use? Do you need to buy a domain for this?

**Comment by heybrodes** [comment]
> What domain do you need to use? Do you need to buy a domain for this?

Yes you need to buy a domian or if you already have one use a subdomain.
